templates/web/kingston/waste/garden/subscribe_intro.html

```diff
@@ -17,14 +17,29 @@ <h2>Costs</h2>
 </p>
 
 <p>
-If you need new bins to be delivered, there is an admin fee of £15 per bin.
+If you need us to deliver bins to you, there is a fee of £15 per bin for admin and delivery. If you order more than one bin at once, this changes to £7.50 per bin.
 </p>
 
+[% IF garden_sacks %]
+<p>
+A roll of 10 garden waste sacks costs £41.
+</p>
+[% END %]
+
 <h2>Delivery time</h2>
 
+[% IF garden_sacks %]
+<p>
+We will deliver your garden waste bin or roll of sacks within 20 working days.
```

Comment:
if it's sacks why are we talking about bins?

Reply:
It turns out properties with refuse sacks can choose between sacks and bins.